exonum-java-binding/packaging/pom.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.exonum.binding</groupId>
+    <artifactId>exonum-java-binding-parent</artifactId>
+    <version>0.5-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>exonum-java-binding-packaging</artifactId>
+  <version>0.5-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <properties>
+    <packaging.basedir>${project.parent.basedir}/core/rust/target/prepackage</packaging.basedir>
+    <packaging.rust>${project.parent.basedir}/core/rust</packaging.rust>
+    <packaging.java>${project.parent.basedir}/core/rust/target/debug/lib/java</packaging.java>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-java-binding-core</artifactId>
+      <version>0.5-SNAPSHOT</version>
+      <type>pom</type>
+    </dependency>
+
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-time-oracle</artifactId>
+      <version>0.5-SNAPSHOT</version>
+      <type>pom</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin><!--Do not create Jar-->
```

Comment:
It is required only if this project uses 'jar' packaging which
binds jar and install plugins by default. But it is no longer the case (it's "pom" now).

I doubt, however, that the "pom" works, because it does not have "package" or "pre-package" phases that are used below. 

I'd add an issue to define a custom packaging for our use-case and *not* abuse the 'jar' packaging. It seems to be quite straightforward: https://stackoverflow.com/questions/1427722/how-do-i-create-a-new-packaging-type-for-maven

Reply:
Thanks, reverted to jar and made task ECR-3039